pages/common/yq.md

```diff
@@ -0,0 +1,37 @@
+# yq
+
+> A lightweight and portable command-line YAML processor.
+> Note that `yq` v3 and v4 do not share commands.
```

Comment:
We use to support only the latest version of a command. 
```suggestion
```

Reply:
I think it makes a lot of sense for `tldr` to support only the most recent release. `yq` 4 was release only 8 days ago and has a completely different UX than `yq` 3 so I felt it was important to keep the continuity. 

https://github.com/mikefarah/yq/releases/tag/4.0.0

Reply:
@navarroaxel are you okay with that?  If so go ahead and merge this.